ToolBox/AlgoSeekOptionsConverter/Program.cs

```diff
@@ -29,72 +32,32 @@ public class Program
 
         static void Main()
         {
-            var directory = "F:/Downloads/AlgoSeek/smaller";
-            directory = "F:/AlgoSeek/20151224";
-            var dataDirectory = "./Data";
+			Environment.SetEnvironmentVariable("MONO_MANAGED_WATCHER", "disabled");
+
+			var sourceDirectory = "/cache/TestData/RawGoog/20151224";
+			var destinationDirectory = "/cache/TestData/Converted";
+
+//			var sourceDirectory = "/cache/DevData/RawDataSmall/20140516";
+//			var destinationDirectory = "/cache/DevData/ConvertedData";
```

Comment:
Please remove commented code & set the source and destination directory to come from config.